uint64 time_info
string left_image_string
string right_image_string